framework/rbac/DbManager.php

```diff
@@ -524,7 +524,7 @@ public function getPermissionsByRole($roleName)
      */
     public function getPermissionsByUser($userId)
     {
-        if (empty($userId)) {
+        if (!is_numeric($userId) && empty($userId)) {
```

Comment:
@jblac Before this PR any non-empty (as numeric and the other) data could be passed to function. Now i added possibility to pass any  numeric data (including empty values that is equal to zero).